images/linux/ubuntu1804.json

```diff
@@ -250,7 +250,8 @@
                 "{{template_dir}}/scripts/installers/yq.sh",
                 "{{template_dir}}/scripts/installers/pypy.sh",
                 "{{template_dir}}/scripts/installers/python.sh",
-                "{{template_dir}}/scripts/installers/aws.sh"
```

Comment:
Push this to line - 254

Reply:
> Push this to line - 254

Do you mean you want the `cassandra.sh` line to appear before `aws.sh` line?

Reply:
Yes.

Reply:
@freddy123098, any particular reason why we would want to do that? Just curious for myself.